### DISCUSSION TITLE
docker环境下如何修改console入口

### DISCUSSION THREAD

**Discussion by foxfire2015** [halo-dev/halo#6422] [Q&A] [answered]
环境：网站映射外网
问题：外网输入/console就可以访问后端管理地址，感觉不安全，万一密码被他碰对了；
解决办法：如何修改后端管理地址，如改为/abc，这样别人不知道，自己有可以外网管理博客文章

**Comment by ruibaby** [comment, marked as answer]
目前不考虑支持，原因可见：https://bbs.halo.run/d/5230-%E7%AE%A1%E7%90%86%E5%90%8E%E5%8F%B0%E5%9C%B0%E5%9D%80/9
